### ISSUE TITLE
Relative pathnames not correctly resolved when used saxon-gradle is used as a sub-project

### ISSUE THREAD

**Issue by AndrewCampbellSynchronoss** [eerohele/saxon-gradle#1]
I have created a simple gradle project that uses saxon-gradle as a sub project - see attached [com.github.eerohele.saxon-gradle.zip](https://github.com/eerohele/saxon-gradle/files/206516/com.github.eerohele.saxon-gradle.zip).

Running `gradle xslt` from the parent directory gives the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':simple:xslt'.
> Fatal error during transformation: java.lang.RuntimeException: Stylesheet file xsl/html5.xsl does not exist
```

Changing `stylesheet` to `simple/xsl/html5.xsl` gives the following error when I run `gradle xslt`:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/acam0006/Desktop/com.github.eerohele.saxon-gradle/simple/build.gradle' line: 38

* What went wrong:
A problem occurred evaluating project ':simple'.
> /Users/acam0006/Desktop/com.github.eerohele.saxon-gradle/simple/simple/xsl/html5.xsl (No such file or directory)

```

Everything works correctly when the absolute path is used. I can also move the xsl directory to the parent directory to get everything to work.

It also seems like an absolute path needs to be used for the value of `dir` in `input`.

I'm using Gradle 2.12

**Comment by eerohele** [comment]
Thanks for the bug report and the sample files!

I've deployed version 0.1.2 of the plugin, which should fix the issue. Can you please give it a go?

Just change:

``` groovy
plugins {
    id 'com.github.eerohele.saxon-gradle' version '0.1.1'
}
```

To:

``` groovy
plugins {
    id 'com.github.eerohele.saxon-gradle' version '0.1.2'
}
```

**Comment by AndrewCampbellSynchronoss** [comment]
That works - thanks for the fast response! 

Now there is an interesting anomaly if the `config` option is used as well. This value has to be set relative to the parent directory, but the `stylesheet` and `input` options are relative to the `simple` directory - see attached: [build.gradle.txt](https://github.com/eerohele/saxon-gradle/files/206635/build.gradle.txt).

**Comment by eerohele** [comment]
> Now there is an interesting anomaly if the config option is used as well. This value has to be set relative to the parent directory, but the stylesheet and input options are relative to the simple directory

Ah, yes, the root cause is basically the same as for `stylesheet` — I've fixed that and released version 0.1.3, which should fix the issue.

**Comment by AndrewCampbellSynchronoss** [comment]
Excellent, that works! Thanks again :+1:
